### PR TITLE
KAA-1277: [C++ SDK] Update C++ build guide for Windows

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @vadimol

### CODE REVIEW

**Comment by vadimol** [comment]
This must be tested.
Overall, looks good for me.

File: client/client-multi/client-cpp/tools/env.bat [hunk 1]
```diff
@@ -40,24 +40,23 @@ if not %BUILD_PLATFORM% == x86 (
   set BUILD_PLATFORM=x64
 )
 
+@REM Root path which will be passed to CMake during SDK compilation
+set ROOT_PATH=C:\local\
```
Comment (GamovCoder) [inline client/client-multi/client-cpp/tools/env.bat]: Consider not using environment variables at all. Probably, all `*bat` files should be dropped. TBD.

File: client/client-multi/client-cpp/tools/build_sdk_thirdparty.bat [hunk 2]
```diff
@@ -79,9 +53,9 @@ goto :eof
   cd %AVRO_SRC%\lang\c++\build.win
 
   if %BUILD_PLATFORM% == x86 (
-    cmake -DCMAKE_INSTALL_PREFIX:PATH=%AVRO_ROOT% -G "Visual Studio %MSVC_VERSION%"  ..
+    cmake -DCMAKE_INSTALL_PREFIX:PATH=%ROOT_PATH% -G "Visual Studio %MSVC_VERSION%"  ..
```
Comment (rasendubi) [inline client/client-multi/client-cpp/tools/build_sdk_thirdparty.bat:56]: If the current build is not broken, I would prefer separate variables for different libraries than a single `ROOT_PATH`

**Comment by GeorgePostolakiy** [comment]
LGTM!

**Comment by vadimol** [comment]
+

**Pull request merged**